T2K 4e/sheet.json

```
@@ -0,0 +1,9 @@
+{
+    "html": "twilight-2k-4e.html",
+    "css": "twilight-2k-4e.css",
+    "authors": "/u/NocFenix(3266353)",
+    "roll20userid": "3266353",
+    "preview": "twilight-2k-4e.png",
+    "instructions": "You can click on the Attribute names to roll against your attributes.",
+    "legacy": false
```

```suggestion
    "legacy": true
```
This is the fix that people are proposing in this thread, as Roll20 haven't yet implemented the new sheet stuff.

You should be able to just approve my edit as is